notifications_api_common/management/commands/register_kanalen.py

```diff
@@ -15,6 +18,27 @@ class KanaalExists(Exception):
     pass
 
 
+class KanaalException(Exception):
+    kanaal: str
+    data: dict | list
+
+    def __init__(self, kanaal: str, data: dict | list = {}):
+        super().__init__()
+
+        self.kanaal = kanaal
+        self.data = data
```

Comment:
Do you really need these parameters? When you catch all kanaal exceptions in the management command below you don't use them for writing in the stderr.